faculty/clients/notification.py

```diff
@@ -126,3 +154,36 @@ def wait_for_completion(self):
                 raise TemplatePublishingError(msg)
             elif e.event == "@SSE/PROJECT_TEMPLATE_PUBLISH_NEW_COMPLETED":
                 return
+
+
+class AddTemplateToProjectFromDirectoryNotifications:
+    """
+    Wrapper around notification events from the "add template to project from
```

Comment:
Python docstrings should start with a single-line description, with further elaboration, if required, put separately in paragraphs, e.g.:

```
"""
A function to do a thing.

Lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum
lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum
lorem ipsum lorem ipsum lorem ipsum.

Lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum
lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum
lorem ipsum lorem ipsum lorem ipsum.

Parameters
----------
...
```